assignment 5/nodecallback.js

```diff
@@ -0,0 +1,38 @@
+var fs = require("fs");
+
+fs.readFile("file1.txt", function(err, data) {
+  if (err) {
+    return console.error(err);
+  }
+  console.log("Reading file 1 complete");
+  writefile2(data);
+});
+
+var writefile2 = function(data) {
+  fs.appendFile("file2.txt", data, function(err) {
+    if (err) {
+      return console.error(err);
+    }
+    console.log("appending in file 2 complete");
+    readFile2();
+  });
+};
+
+var readFile2 = function() {
+  fs.readFile("file2.txt", function(err, data) {
+    if (err) {
```

Comment:
@abansal26 Can we make separate function for handling the error ?

Reply:
Yes we can but it will be of no use as we will have to make another function to process data which will have both need error as well as data to process and as error we will pass null

Reply:
Basically fs.readFile("file2.txt", function(err){.........},function(null, data){.......})